config/config/sc-config.yaml

```diff
@@ -34,6 +34,7 @@ user:
     tolerations: []
     affinity: {}
     nodeSelector: {}
+    extraConfigmapMounts: []
```

Comment:
I'd actually think it'd be great if these had an example of how to mount the ca-certificate.